static/js/chart/draw_bivariate.ts

```diff
@@ -97,10 +97,9 @@ function drawMapLegend(
   xUnit?: string,
   yUnit?: string
 ): void {
-  const legendSvg = d3
-    .select(legendRef.current)
-    .append("svg")
-    .attr("width", svgWidth);
+  const container = d3.select(legendRef.current)
+  container.selectAll("*").remove();
```

Comment:
what's the effect of the change here?

Reply:
this fixes a bug in the bivariate tile where multiple legends were showing up because old legend was not getting removed